x/stakeibc/keeper/hooks.go

```diff
@@ -184,46 +184,42 @@ func (k Keeper) UpdateRedemptionRates(ctx sdk.Context, depositRecords []recordst
 	}
 }
 
-// Determine the undelegated balance from the deposit records queued for staking
-func (k Keeper) GetUndelegatedBalance(hostZone types.HostZone, depositRecords []recordstypes.DepositRecord) sdk.Dec {
-	// filter to only the deposit records for the host zone with status DELEGATION_QUEUE
-	undelegatedDepositRecords := utils.FilterDepositRecords(depositRecords, func(record recordstypes.DepositRecord) (condition bool) {
-		return ((record.Status == recordstypes.DepositRecord_DELEGATION_QUEUE ||
-			record.Status == recordstypes.DepositRecord_DELEGATION_IN_PROGRESS) &&
-			record.HostZoneId == hostZone.ChainId)
-	})
-
-	// sum the amounts of the deposit records
+// Determine the deposit account balance, representing native tokens that have been deposited
```

Comment:
"deposited from liquid stakes" sounds like the opposite thing it sounds like they are handing you a liquid stake for redemption.   I would say "native tokens that have been deposited to create new liquid stakes, ..."

Reply:
A liquid stake is a transaction - so this is saying deposits from the transaction.

The user doesn't "create a liquid stake" (noun), they "liquid stake" (verb)